### ISSUE TITLE
Shortcodes not working

### ISSUE THREAD

**Issue by shivampaw** [Automattic/amp-wp#458]
Hi,

I have a custom shortcode for a click to tweet link.

```
function click_to_tweet_shortcode( $atts, $content = "" ) {
	$url = 'https://twitter.com/intent/tweet?text='.$content.'&url='.get_permalink().'&via=depression_911';
	return '<blockquote><p>'.$content.' - <a target="_blank" href="'.$url.'" title="'.$content.'">Click to Tweet</a></p></blockquote>';
}
add_shortcode( 'ctt', 'click_to_tweet_shortcode' );
```
It works fine on the normal version: https://www.depressionforteens.com/you-should-talk-about-your-depression/ 

But not on AMP: https://www.depressionforteens.com/you-should-talk-about-your-depression/amp/

The example is at the bottom of the post.

Thanks

**Comment by mjangda** [comment]
Please see https://github.com/Automattic/amp-wp#handling-media for handling media like shortcodes.

**Comment by shivampaw** [comment]
@mjangda It isn't making much sense to me - if it's "unaffected" then how come it isn't showing?

**Comment by mjangda** [comment]
![you_should_talk_about_your_depression_-_here_s_why_](https://cloud.githubusercontent.com/assets/86105/18065411/cdeb2100-6e01-11e6-8e9e-35eb1a665e2d.png)

I see the content showing up when I view the AMP version. Why the link doesn't work might be fixed with #449

**Comment by shivampaw** [comment]
Yeah the problem is with the link. I'll wait for the merge and update then I guess.
